Specs/Number/Arabic/NumberRangeModel.json

```diff
@@ -93,7 +93,7 @@
         "Text": "أكبر من ١٠٠ وأصغر من ٣٠٠",
         "TypeName": "numberrange",
         "Resolution": {
-          "value": "-100,300"
+          "value": "(100,300)"
```

Comment:
This sentence was not with -100? Do we have one with it to add too?

Reply:
Sure! will add into next iteration.